packages/widgetbook/lib/src/addons/device_addon/device_addon.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/src/addons/device_addon/frames/frames.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_core/widgetbook_core.dart';
+
+class DeviceAddon extends WidgetbookAddOn<DeviceSetting> {
+  DeviceAddon({
+    required List<Device> devices,
+    Device? initialDevice,
+  })  : assert(
+          devices.isNotEmpty,
+          'Please specify at least one Device',
+        ),
+        assert(
+          devices.contains(initialDevice),
+          '[initialDevice] must be in [devices]',
```

Comment:
Will this generate a proper error message? I'm assuming it'll show  `[initialDevice] must be in [devices]` instead of the intended `initialDevice must be in devices`, but maybe that's a cool flutter feature I didn't know about